src/Fabulous/Program.fs

```diff
@@ -3,12 +3,19 @@ namespace Fabulous
 open System
 open System.Diagnostics
 
+(*TODO Is either of these a program in the Elm sense? If so, where's the view in this one?
```

Comment:
Originally we had a single `Program` type containing the `view` function like in typical Elm. In Fabulous 3, I had to split the Program type in 2 types to enable support of MVU components because the view function is defined later.

```fs
let init = ...
let update = ...
let view = ...

let program =
    Program.stateful init update // Program<'arg, 'model, 'msg>
    |> Program.withView view // Program<'arg, 'model, 'msg, 'view>
``` 

```fs
let init = ...
let update = ...
let program = Program.stateful init update

let view =
    Component(program) {
        // the actual view function here
    }
```

Reply:
Do you think it would be better to rename the two Program types to avoid the confusion?